lisa/tools/ethtool.py

```diff
@@ -221,6 +221,15 @@ def _parse_ring_buffer_settings_info(self, interface: str, raw_str: str) -> None
             )
 
 
+class DeviceGroLroSettings:
+    def __init__(
+        self, interface: str, gro_setting: str = "off", lro_setting: str = "off"
```

Comment:
Use True, False is easier to write logic, and avoid typo. They can be translated to "on" and "off" when using in command.